maplibre_gl_example/pubspec.yaml

```diff
@@ -26,6 +26,12 @@ dev_dependencies:
     sdk: flutter
   very_good_analysis: ^5.0.0
 
+dependency_overrides:
+  maplibre_gl_platform_interface:
+    path: ../maplibre_gl_platform_interface
+  maplibre_gl_web:
+    path: ../maplibre_gl_web
+
```

Comment:
Those dependency overrides shouldn't be necessary when resolving the dependencies with melos.